maven-plugin/src/main/resources/help.yaml

```diff
@@ -36,6 +48,10 @@ help:
         desc: "Artifact id of an artifact, which you want to deploy."
       - name: "Dversion"
         desc: "Version of an artifact, which you want to deploy."
+      - name: "Ddistro"
+        desc: "OpenMRS Distribution to set up in the format 'groupId:artifactId:version'. You can skip groupId, if it is 'org.openmrs.distro'. You can also give Path to distro.properties file"
+      - name: "Dowa"
+        desc: "Owa property in the format 'owa-name:version'"
```

Comment:
Which property? Version or something else?

Reply:
I found the template for this parameter from here: https://github.com/openmrs/openmrs-sdk/blob/12d3ba529b963ab1e8b81e55ce195e883d8facb1/maven-plugin/src/main/java/org/openmrs/maven/plugins/Deploy.java#L113-L119